frontend/www/js/omegaup/components/badge/Badgev2.test.ts

```diff
@@ -0,0 +1,17 @@
+import { shallowMount } from '@vue/test-utils';
+import expect from 'expect';
+import omegaup_badge from './Badgev2.vue';
```

Comment:
```suggestion
import badge_Badge from './Badgev2.vue';
```

por consistencia con los demás lugares donde se usa este import.

Reply:
Entiendo, pero no sería en todo caso ?
```suggestion
import Badge from './Badgev2.vue';
```
por que así aparece en `badge/List.vue`

Reply:
oops... https://github.com/omegaup/omegaup/pull/4174/

_ahora_ sí.